docs/src/eval.md

```diff
@@ -6,14 +6,21 @@ Given an expression tree specified with a `Node` type, you may evaluate the expr
 over an array of data with the following command:
 
 ```@docs
-eval_tree_array(tree::Node{T}, cX::AbstractMatrix{T}, operators::OperatorEnum) where {T<:Number}
+eval_tree_array(
+    tree::AbstractExpressionNode{T},
+    cX::AbstractMatrix{T},
+    operators::OperatorEnum;
+    eval_options::Union{EvalOptions,Nothing}=nothing,
+    turbo::Union{Bool,Val,Nothing}=nothing,
+    bumper::Union{Bool,Val,Nothing}=nothing,
+) where {T}
```

Comment:
```suggestion
eval_tree_array
```

(Should still work)

Reply:
Yes, it does, but then it also shows the other docstring for `GenericOperatorEnum`. It seemed like this was selecting only the `OperatorEnum` one on purpose

Reply:
Oh, right! Sorry about that.